interface/ceed.c

```diff
@@ -843,6 +843,7 @@ int CeedInit(const char *resource, Ceed *ceed) {
     CEED_FTABLE_ENTRY(CeedVector, SetArray),
     CEED_FTABLE_ENTRY(CeedVector, TakeArray),
     CEED_FTABLE_ENTRY(CeedVector, SetValue),
+    CEED_FTABLE_ENTRY(CeedVector, SyncArray),
```

Comment:
`SyncArray` was already part of the `CeedVector_private` struct's list of backend functions, but it had never been added to the table here